lib/reporters/teamcity_reporter.js

```diff
@@ -38,7 +38,9 @@ TeamcityReporter.prototype = {
     if (result.skipped) {
       this.out.write('##teamcity[testIgnored name=\'' + this._namify(prefix, result) + '\' message=\'pending\']\n');
     } else if (!result.passed) {
-      this.out.write('##teamcity[testFailed name=\'' + this._namify(prefix, result) + '\' message=\'' + escape(result.error.message) + '\' details=\'' + escape(result.error.stack) + '\']\n');
+      var message = (result.error && result.error.message) || '';
+      var stack = (result.error && result.error.stack) || '';
+      this.out.write('##teamcity[testFailed name=\'' + this._namify(prefix, result) + '\' message=\'' + escape(message) + '\' details=\'' + escape(stack) + '\']\n');
```

Comment:
No what I meant, wouldn't it be better to omit those properties when they are empty instead of setting them to an empty string?